apps/edxapp/vars/all/main.yml

```diff
@@ -67,6 +67,7 @@ edxapp_data_volume_size: 2Gi
 edxapp_media_volume_size: 2Gi
 edxapp_static_volume_size: 2Gi
 edxapp_locale_volume_size: 200Mi
+edxapp_exports_volume_size: 100Mi
```

Comment:
Are you sure about this tiny size?

Reply:
yes, for now. those files have short livetime

Reply:
ok then.